tensorboard/plugins/profile/tf_op_profile/tf-op-table.html

```diff
@@ -74,6 +111,20 @@
 /* Match indented names */
 #header > #name { padding-left: 2em; }
     </style>
+    <div id="control">
+    <!--
+      paper-slider sets how many child nodes to show for each category.
+      If paper-toggle-button is checked, then only children in top 90th
+      percentile of time spent are listed.
+    -->
+    <span class="controlRowLeft">Show top
+      <paper-slider min="10" max="100" snaps step="10"
+        value="{{childrenCount}}" editable>
+      </paper-slider>ops</span>
+    <span class="controlRowRight">off&nbsp
```

Comment:
Should there be a semicolon at the end? ie, `&nbsp;`

Reply:
Done!